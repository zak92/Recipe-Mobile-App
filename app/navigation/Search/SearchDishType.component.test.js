
import React from 'react';
import { render } from '@testing-library/react-native'; // json rendering
import SearchDishType from './SearchDishType.component';


describe('SearchDishType ', () => {
  it("should match snapshot", () => {
    const snapSearchDishType = render(<SearchDishType />).toJSON(); // render this file to JSON
    expect(snapSearchDishType).toMatchSnapshot(); // this snapshot should match the previous snapshot

  });
  it('displays the passed-in dishType', () => {
    const { queryByText } = render(
      <SearchDishType dishType="Main course" />
    );
    expect(queryByText('Main course')).not.toBeNull();
  });


});
