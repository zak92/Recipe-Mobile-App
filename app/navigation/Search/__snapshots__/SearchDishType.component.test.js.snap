// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchDishType  should match snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
      "width": "10%",
    }
  }
>
  <Image
    resizeMode="cover"
    style={
      Object {
        "borderRadius": 5,
        "height": 60,
        "width": 60,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#12ad36",
        "padding": "8%",
      }
    }
    testID="dishType"
  />
</View>
`;
